src/styles/components/index.scss

```diff
@@ -0,0 +1,33 @@
+.home {
+  position: relative;
+}
+
+.home-top {
+  padding: 13rem 0.5rem 0 8rem;
+  max-width: 47%;
+
+  h1 {
+    font-size: 2.5rem;
+  }
+
+  &__top-description {
+    font-size: 1.1rem;
+  }
+
+  &__top-image {
+    position: absolute;
+    right: 1rem;
+    top: 1rem;
+  }
+}
+
+@media (max-width: 1258px) {
```

Comment:
What's the value of `1258` coming form? Is that an agreed-upon value for a certain screen size?